router/types/types.go

```diff
@@ -11,10 +11,7 @@ import (
 	"github.com/rudderlabs/rudder-server/jobsdb"
 )
 
-const (
-	RouterUnMarshalErrorCode = 599
-	RouterTimedOutStatusCode = 1113
-)
+const RouterUnMarshalErrorCode = 599
```

Comment:
Can use the string type considering the fact that we're converting to string everywhere